_includes/v21.2/misc/debug-subcommands.md

```diff
@@ -1,3 +1,5 @@
 While the `cockroach debug` command has a few subcommands, users are expected to use only the [`zip`](cockroach-debug-zip.html), [`encryption-active-key`](cockroach-debug-encryption-active-key.html), [`merge-logs`](cockroach-debug-merge-logs.html), [`list-files`](cockroach-debug-list-files.html), and [`ballast`](cockroach-debug-ballast.html) subcommands.
 
+The [`job-trace`](cockroach-debug-job-trace.html) subcommand, while applicable for users, we recommend for use only when directed by the Cockroach Labs support team.
```

Comment:
I feel like this is misleading.  If we are putting it in the docs, we are de facto saying it is for users.

Since that ship is already underway ... :-) 

my comment is: I would find this easier to understand if the clauses were reversed and the "while applicable for users" were removed, e.g.,

"We recommend using the `job-trace` command only when directed by the Cockroach Labs support team"

and then making 'support team' a link to our support page at https://www.cockroachlabs.com/docs/v21.2/support-resources.html

also, I noticed it was in a callout on the full docs page below.  may be worth putting it in a callout here too